addons/rose/addon/styles/rose/components/form/_radio.scss

```diff
@@ -21,6 +21,12 @@ $radio-svg: (
 );
 
 $data-svg-prefix: 'data:image/svg+xml;utf-8,';
+.radio-group-inline {
```

Comment:
my bad, this is incorrect. This should be `inline` which comes from `addons/rose/addon/components/rose/form/radio/radio/index.hbs` line 2